tar_scm.py

```diff
@@ -54,6 +54,10 @@ def run_cmd(cmd, cwd, interactive=False, raisesysexit=False):
     env = os.environ.copy()
     env['LANG'] = 'C'
 
+    if is_proxy_defined():
+        env['http_proxy'] = os.environ.get('http_proxy')
```

Comment:
OK, that's a good reason. Although it's disappointing that the `curl` and `profile.sh` authors decided to deviate from a convention widely accepted since 1979.